cypress/e2e/seededFlows/moderationFlows/adjustPostTags.spec.js

```diff
@@ -1,383 +1,388 @@
+//         cy.findByTestId('add-tag-button').click();
+//         cy.findByPlaceholderText('Add a tag').type('tag2');
+//         cy.findByPlaceholderText('Reason to add tag (optional)').type(
+//           'testing',
+//         );
+//         cy.findByRole('button', { name: 'Add tag' }).click();
+//       });
+
+//       cy.getIframeBody('.article-iframe').within(() => {
+//         cy.findByText('The #tag2 tag was added.');
+//         cy.findByRole('link', { name: /# tag2/ });
+//       });
+//     });
+
+//     it('should remove a tag from a post', () => {
+//       cy.findByRole('heading', { name: 'Tag test article' }).click();
+//       cy.getIframeBody('.article-iframe').findByRole('link', {
+//         name: /# tag1/,
+//       });
+
+//       cy.getIframeBody('.actions-panel-iframe').within(() => {
+//         cy.findByRole('button', { name: 'Open adjust tags section' }).click();
+//         cy.findByText('tag1').click();
+//         cy.findByPlaceholderText('Reason to remove tag (optional)').type(
+//           'testing',
+//         );
+
+//         cy.findByRole('button', { name: 'Remove tag' }).click();
+//       });
+
+//       cy.getIframeBody('.article-iframe').within(() => {
+//         cy.findByText('The #tag1 tag was removed.');
+//         cy.findByRole('link', { name: /# tag1/ }).should('not.exist');
+//       });
+//     });
+
+//     it('should show previous tag adjustments', () => {
+//       cy.intercept('/tag_adjustments').as('tagAdjustmentRequest');
+//       cy.findByRole('heading', { name: 'Tag test article' }).click();
+//       // cy.findByRole('main').as('main');
+
+//       cy.getIframeBody('.actions-panel-iframe').within(() => {
+//         // Click listeners are attached async so we use pipe() to retry click until condition met
+//         cy.findByRole('button', {
+//           name: 'Open adjust tags section',
+//         })
+//           .pipe(click)
+//           .should('have.attr', 'aria-expanded', 'true');
+
+//         cy.findByTestId('add-tag-button').click();
+//         cy.findByPlaceholderText('Add a tag').type('tag2');
+//         cy.findByPlaceholderText('Reason to add tag (optional)').type(
+//           'testing',
+//         );
+//         cy.findByRole('button', { name: 'Add tag' }).click();
+//       });
+
+//       cy.wait('@tagAdjustmentRequest');
+
+//       cy.getIframeBody('.actions-panel-iframe').within(() => {
+//         cy.findByRole('button', {
+//           name: 'Open adjust tags section',
+//         })
+//           .pipe(click)
+//           .should('have.attr', 'aria-expanded', 'true');
+
+//         cy.findByText('tag1').click();
+//         cy.get('#tag-removal-reason-tag1').type('testing');
+
+//         cy.findByRole('button', { name: 'Remove tag' }).click();
+//       });
+
+//       cy.wait('@tagAdjustmentRequest');
+//       // these reloads 'make the test work for now' but inconsistently,
```

Comment:
added this note here so the reminder will be commented above the first usage of reload